### PR TITLE
Add OpenAI models from Jan 25th

### PR DESCRIPTION
**Pull request by philippzagar**
## What

Update the OpenAI library with the newest released [OpenAI models from January 25th](https://openai.com/blog/new-embedding-models-and-api-updates).
Most importantly, the PR introduces support for the `gpt-4-turbo-preview` (alias for `gpt-4-0125-preview`) and `gpt-3.5-turbo-0125` (aliased by `gpt-3.5-turbo`) models.

## Why

Developers are now able to use the newest models within their applications.

## Affected Areas

Only minor adjustments within the `Model.swift` file, as well as the README.

### CODE REVIEW

**Comment by philippzagar** [comment]
@Krivoblotsky @ingvarus-bc @SunburstEnzo Would be great to get this out quickly so that people can start using the new models, the PR contains only minor code changes 🚀

File: README.md
Comment (SunburstEnzo) [inline README.md:803]: Perhaps this should be `gpt3_5Turbo_0613` like the others (same for 0301), but not a deal breaker 👍

**Review by SunburstEnzo — APPROVED** [review]

**Comment by ingvarus-bc** [comment]
Hi, @philippzagar, thanks for your PR! 
It seems like moderations model has been changed and now tests for it are failing. Perhaps changes to moderations from [this PR](https://github.com/MacPaw/OpenAI/pull/146) could fix it? 🤔

**Review by ingvarus-bc — APPROVED** [review]

File: README.md
Comment (philippzagar) [reply README.md]: Good catch @SunburstEnzo, code is adjusted! 🚀

**Comment by philippzagar** [comment]
@ingvarus-bc @SunburstEnzo Please approve the workflow execution once again and then we can merge the PR 🚀

**Review by SunburstEnzo — APPROVED** [review]

**Comment by philippzagar** [comment]
@ingvarus-bc Would you mind tagging a new release?

**Comment by philippzagar** [comment]
@ingvarus-bc @SunburstEnzo As tons of new changes have been merged, would you mind tagging a new release for the library?

**Comment by jagilley** [comment]
Yes please, +1 on tagging a new release @ingvarus-bc

**Comment by philippzagar** [comment]
@ingvarus-bc @SunburstEnzo @jagilley Created a new issue to track the new release: https://github.com/MacPaw/OpenAI/issues/171

**Pull request merged**
